open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-api/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/event/VirtualizerTopicListener.java

```diff
@@ -59,10 +59,12 @@ public void processEvent (String event){
         ObjectMapper objectMapper = new ObjectMapper();
         try{
             TableContextEvent eventObject = objectMapper.readValue(event, TableContextEvent.class);
-            Map<String, String> views = viewGeneratorConnector.processInformationViewEvent(eventObject);
-            List<DataPlatformEvent> viewEvents = generateViewEvents(eventObject, views);
-            for (DataPlatformEvent item : viewEvents){
-                ivInTopicConnector.sendEvent(objectMapper.writeValueAsString(item));
+            if (eventObject != null){
+                Map<String, String> views = viewGeneratorConnector.processInformationViewEvent(eventObject);
+                List<DataPlatformEvent> viewEvents = generateViewEvents(eventObject, views);
+                for (DataPlatformEvent item : viewEvents){
+                    ivInTopicConnector.sendEvent(objectMapper.writeValueAsString(item));
```

Comment:
Use generic variable names as this is no longer inTopic. Please note also configuration should be changed, names like ivInTopicName and ivOutTopicName should be generic.